bin/p2-label/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/square/p2/pkg/kp"
+	"github.com/square/p2/pkg/kp/flags"
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/util"
+	"gopkg.in/alecthomas/kingpin.v2"
+	klabels "k8s.io/kubernetes/pkg/labels"
+)
+
+var (
+	cmdApply               = kingpin.Command(CmdApply, "Apply label changes to all objects matching a selector")
+	applyLabelType         = cmdApply.Flag("labelType", "The type of label to adjust. Sometimes called the \"label tree\"").Short('t').Required().String()
```

Comment:
You could consider looping through the label constants in the help message https://github.com/square/p2/blob/master/pkg/labels/applicator.go#L23-L29